src/js/modal.js

```diff
@@ -66,6 +76,36 @@ export function imageButtonClick(movie) {
 
   window.addEventListener('keydown', handleEvent);
 
+  // trailer -->
+  function playTrailer() {
+    fetchID(movie.id, '/videos')
```

Comment:
W drugim argumencie wystarczyłoby samo true, ale funkcja jest jednak na tyle generyczna, że cokolwiek się wstawi to wrzuci właściwy sufix :)